src/rt/sections_android.d

```diff
@@ -180,9 +183,11 @@ extern(C)
         void* __start_minfo;
         void* __stop_minfo;
 
+        void* _tlsend;
         size_t __bss_end__;
 
+    }extern
```

Comment:
What's going on here?

Reply:
Avoids linker issues related to emulated TLS, just a hack for now.